<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap" />
  </node>
  <node name="keypoint_detector" pkg="object_detection" type="keypoint_detector_node" output="screen" args="SURF SURF BruteForce CrossCheckFilter $(find object_detection)/testdata/namaste.jpg 3">
    <remap from="image" to="/usb_cam/image_raw" />
  </node>
</launch>

